### DISCUSSION TITLE
HTTPScaledObject stuck in Pending Creation

### DISCUSSION THREAD

**Discussion by vidyashankar13** [kedacore/http-add-on#433] [General]
Hi, 

I am trying to scale one of our applications based on HTTP requests, I have followed all steps mentioned in install /walkthrough doc. Once my scaled object is created the status of my HTTPScaledObject shows Pending creation for long time and we are unable to test the scaling . We checked the crd's (httpsscaledobject.keda.sh, scaled object.keda.sh) and everything looks fine.

Are we missing anything?
k describe HTTPScaledObject myapp:
<img width="465" alt="so" src="https://user-images.githubusercontent.com/50440333/164190235-0525a11a-8830-4088-846b-9d872822cdd0.PNG">

Update: For the host we have given a ALB URL with path and we saw errors from Keda Operator that ALB domain with / is invalid. 

Is there a way that we can provide a path ?

**Comment by arschles** [comment]
> We checked the crd's (httpsscaledobject.keda.sh, scaled object.keda.sh) and everything looks fine.

@vidyashankar13 when you say you checked the CRDs, do you mean a new `ScaledObject` instance of the CRD has been created? Also, can you get the logs of the KEDA HTTP operator pod? That pod will be named something similar to this: `keda-add-ons-http-controller-manager-688d784b97-qnhmr`

**Comment by linkvt** [comment]
Closing as outdated — this predates several releases and either got resolved since or never reached a conclusion. See current docs (https://keda.sh/http-add-on/0.15/) or feel free to open a new discussion if still relevant.
